src/components/Square.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { getByRole, logRoles, render, screen } from '@testing-library/react';
+import Square from './Square';
+import setup from '../userEvent';
+
+describe('Square component', () => {
+    it('should call function onClick', async () => {
+        const mockFunction = jest.fn();
+        const { user } = setup(
+            <Square
+                value='O'
+                onSquareClick={() => {
+                    mockFunction();
+                }}
```

Comment:
suggestion: 以下でも良きです

```suggestion
                onSquareClick={mockFunction}
```